src/components/profile/edit/EditAccounts.vue

```diff
@@ -36,7 +36,26 @@
         ]"
         v-model="urisUpdated.values[index].k"
       />
-      <input type="text" v-model="urisUpdated.values[index].v" />
+      <label :for="`field-account-${index}-username`" class="visually-hidden"
+        >Username on
+        {{
+          `${
+            EXTERNAL_ACCOUNTS[destructUriKey(urisUpdated.values[index].k).name]
+              .text
+          }`
+        }}</label
+      >
+      <input
+        :id="`field-account-${index}-username`"
+        type="text"
+        v-model="urisUpdated.values[index].v"
+        :placeholder="
+          `Your username on ${
+            EXTERNAL_ACCOUNTS[destructUriKey(urisUpdated.values[index].k).name]
```

Comment:
replace `EXTERNAL_ACCOUNTS[destructUriKey(urisUpdated.values[index].k).name]` with `EXTERNAL_ACCOUNTS[destructUriKey(k).name]`.
We only need the long one when binding via `v-mode` afaik.